lib/pengine/native.c

```diff
@@ -1341,7 +1341,7 @@ pe__native_is_filtered(pe_resource_t *rsc, GList *only_rsc, gboolean check_paren
     if (pcmk__str_in_list(only_rsc, rsc_printable_id(rsc), pcmk__str_none) ||
         pcmk__str_in_list(only_rsc, rsc->id, pcmk__str_none)) {
         return FALSE;
-    } else if (check_parent) {
+    } else if (check_parent && rsc->parent && rsc->parent->variant != pe_native) {
```

Comment:
Only rsc->parent is necessary ... rsc->parent is NULL for bare primitives, it's uber_parent(rsc) that's rsc

Reply:
Fixed.  I think I thought that rsc->parent would just point right back to rsc in this case.